### PR TITLE
Movie sheet

### PR DESCRIPTION
**Pull request by Quentin-Macquart**

### CODE REVIEW

**Review by BenZen59 — DISMISSED** [review]

File: src/components/FicheFilm/FicheFilm.css [hunk 1]
```diff
@@ -1,109 +1,93 @@
...
-
-.overview {
-  font-size: 25px;
-  text-align: left;
-  margin-top: 10px;
-  margin-right: 30%;
-  width: 60%;
-}
-
 .titreVote {
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.css]: Franglish ???

File: src/components/FicheFilm/FicheFilm.jsx [hunk 2]
```diff
@@ -1,138 +1,142 @@
...
-  const [voteAverages, setVoteAverages] = useState([]);
-  const [numberVotes, setNumberVotes] = useState([]);
-  const [posters, setPosters] = useState([]);
-  const [backdrops, setBackdrops] = useState([]);
-  const [directors, setDirectors] = useState([]);
-  const [trailers, setTrailers] = useState([]);
-  const [overviews, setOverviews] = useState([]);
-  const [runtimes, setRuntimes] = useState([]);
-  const [releaseDates, setReleaseDates] = useState([]);
+  const [title, setTitle] = useState([]);
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: an array must be on plural (the same for all your states)

File: src/components/FicheFilm/FicheFilm.jsx [hunk 3]
```diff
@@ -1,138 +1,142 @@
...
 
-  const hour = (runtimes - (runtimes % 60)) / 60;
-  const min = (runtimes / 60 - hour) * 60;
+  const hour = (runtime - (runtime % 60)) / 60;
+  const min = (runtime / 60 - hour) * 60;
 
   return (
-    <div>
+    <div className="filmMainBloc">
       <div className="titreVote">
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: franglish ???

File: src/components/FicheFilm/FicheFilm.jsx [hunk 4]
```diff
@@ -1,138 +1,142 @@
...
+          <br />
+          <div className="numberVote">{numberVote} votes</div>
+        </p>
       </div>
       <div className="posterTrailer">
         <img
           className="backgroundPoster"
-          src={`https://image.tmdb.org/t/p/original${backdrops}`}
+          src={`https://image.tmdb.org/t/p/original${backdrop}`}
           alt="fond"
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: is it in english or french ?

File: src/components/FicheFilm/FicheFilm.jsx [hunk 5]
```diff
@@ -1,138 +1,142 @@
...
+          <p className="overview">{overview}</p>
         </div>
       </div>
-
+      <div className="buttons">
+        <button type="button" className="buttonAdd">
+          +
+        </button>
+        <a href={`https://www.youtube.com/embed/${trailer.key}`}>
+          <div className="buttonTrailer" alt="Trailer">
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: alt on camelCase like the others

**Review by Hacene010 — CHANGES_REQUESTED** [review]
seen some evils

**Review by Othmane-Khiter — APPROVED** [review]
ok

**Review by Hacene010 — APPROVED** [review]
that's better

**Pull request merged**